### DISCUSSION TITLE
[mini.ai] `ci"` doesnt work as expected

### DISCUSSION THREAD

**Discussion by barraIhsan** [nvim-mini/mini.nvim#1252] [Q&A]
When I do `ci"` using `mini.ai`, they will ignore escaped quote and treat them like the actual quote

https://github.com/user-attachments/assets/1c1e11cd-cbc2-4249-ac1d-81ade09b89e5

And this is the vanilla `ci"` (`nvim --clean`)

https://github.com/user-attachments/assets/f8c43998-8946-46cc-8b60-bce15b3ec858

I also tried disabling `mini.ai`, and it work as intended

```lua
{
  "echasnovski/mini.nvim",
  version = "*",
  config = function()
    -- a/i textobjects
    require("mini.ai").setup()
  end,
}
```

**Comment by echasnovski** [comment]
Yes, I am afraid this is how 'mini.ai' works: it treats any `"` as a valid `"` (and tries to find [balanced quotes](https://github.com/echasnovski/mini.nvim/blob/848c5e8f428faf843051768e0d56104cd02aea1f/doc/mini-ai.txt#L23), for that matter). This decision is mostly due to a combination of willing to have a more consistent behavior across all built-in textobjects and Lua patterns implementation details (in particular relying on `%bxy` balanced pattern).

If you want the built-in behavior for `"` and don't mind loosing the "next"/"prev" forced search methods together with consecutive application in Visual mode, you can disable 'mini.ai' only for `"` [by setting it to `false` in `config.custom_textobjects`](https://github.com/echasnovski/mini.nvim/blob/848c5e8f428faf843051768e0d56104cd02aea1f/doc/mini-ai.txt#L479-L480).
